scripts/src/index.ts

```diff
@@ -0,0 +1,199 @@
+import fs from "fs";
+import path from "path";
+import matter from "gray-matter";
+import algoliasearch from "algoliasearch";
+
+import simulationMap from "../../resources/docs/url_map.json";
+
+function uuidv4() {
+  return "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, function (c) {
+    var r = (Math.random() * 16) | 0,
+      v = c == "x" ? r : (r & 0x3) | 0x8;
+    return v.toString(16);
+  });
+}
+
+const addUuidGlossary = () => {
+  const files = fs.readdirSync("../resources/glossary");
+
+  for (const fileName of files) {
+    const file = fs.readFileSync(`../resources/glossary/${fileName}`, "utf8");
+
+    fs.writeFileSync(
+      `./output/glossary/${fileName}`,
+      file.replace("---", `---\nobjectId: ${uuidv4()}`)
+    );
+  }
+};
+
+const getAllFiles = function (
+  dirPath,
+  arrayOfFiles: Array<{ inputPath: string; outputPath: string }>
+) {
+  const files = fs.readdirSync(dirPath);
+
+  arrayOfFiles = arrayOfFiles || [];
+
+  files.forEach(function (file) {
+    if (file === "LICENSE.MD") {
+      return;
+    }
```

Comment:
Can we move the license file to the root of `resources` to avoid having to do this? 

I assume we want the CC license across all documentation cc @nonparibus

Reply:
CC BY-SA 4.0, yes.